TurkishFilter/sections/specific.txt

```diff
@@ -43,6 +43,7 @@ fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfi
 !
 temizsozluk.com.tr##a[rel^="sponsored"]
 temizsozluk.com.tr##section > [class*="mx-auto"][class*="text-center"][class*="text-xs"]
+||wishjus.com/t.js$third-party,domain=asistanaliz.com
```

Comment:
to turkish adservers I suppose
`||wishjus.com^$third-party`

Reply:
Okay as you wish.

Reply:
Or maybe better to Base filter?
I saw the same/similar script on one of the Polish website - https://github.com/AdguardTeam/AdguardFilters/commit/029c755967e87ae671fbe1b9ac1b59f3954860ad#diff-fbf56167647d1d9892a9a9ccdd71a6c88bdf70c0d33822f99e0270e936c4b898R6702

Reply:
@AdamWr Okay. Can you please add it directly?

Reply:
Thank you.

Reply:
Done - https://github.com/AdguardTeam/AdguardFilters/commit/3336fa0cf4e183c8a6030deb967a7bcc29b8927a